Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM golang:1.18
```

Comment:
use current Go version?

Reply:
As a library we use the "lowest allowed version" here, which is 1.18 where generics were introduced. We don't want to force clients to use the latest version and we need to make sure the process works with 1.18

Reply:
i dont get it :) what has a docker file go version to do with the go version our sdk clients will use? :)